src/components/socio-carddesktop.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+
+const ColaboradorCardDesktop: React.FC = () => {
+  return (
+    <div className="relative flex h-[192px] w-full max-w-[1300px] items-center overflow-hidden rounded-xl bg-blue-600 p-6 text-white">
+      <div className="absolute bottom-0 left-0 h-[87px] w-[148px] md:h-[148px] md:w-[236px]">
+        <img
+          src="/sociocard.svg"
+          alt="Colaborador Icon"
+          className="h-full w-full object-contain"
+        />
+      </div>
+      <div className="ml-[236px] flex-grow">
+        <h4 className="font-orbitron mb-2 text-3xl">Torna-te colaborador</h4>
+        <p className="font-inter text-base">
+          Junta-te a um (ou vários) departamentos e ajuda a tornar o CeSIUM
```

Comment:
This text isn't matching the one in the design 🤔 

![image](https://github.com/user-attachments/assets/f57f2dcb-e71c-469f-a375-845ea25a8407)